portal/tests/utils/messages.py

```diff
@@ -48,6 +48,18 @@ def is_teacher_details_updated_message_showing(browser):
     return is_message_showing(browser, 'Your account details have been successfully changed.')
 
 
+def is_student_details_updated_message_showing(browser):
+    return is_message_showing(browser, 'Your account details have been changed successfully.')
```

Comment:
That might be a better idea to check if the message showing is the actual message in whatever messages.py we pick the line being sent to students? Here we could introduce spelling mistakes for instance. Also, localisation.